config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get "/merchants/:id/dashboard", to: "merchants#show"
+  # get "/merchants/:id/items", to: "items#show"
+  resources :merchants do
```

Comment:
we also have a similar route but ours links to a merchant_items controller. Not sure if that will be too conflicting but can discuss later